src/Components/Navbar.js

```diff
@@ -1,9 +1,15 @@
 import React from 'react';
 import { NavLink } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import { getMissions } from '../redux/missions/missions';
 import icon from '../img/icon.png';
 import '../css/Navbar.css';
 
 export default function Navbar() {
+  const dispatch = useDispatch();
+  const fetchDataMissions = () => {
+    dispatch(getMissions());
+  };
   return (
     <nav className="flex">
       <div className="logo flex">
```

Comment:
[OPTIONAL]: As a recommendation, It would be better if you create a separate file for Missions (i.e. /Components/MissionsContainer) and  then you call it inside pages (/pages/Missions)

import MissionsContainer from '../Components/MissionsContainer';

const Missions = () => (
  <div>
    <MissionsContainer />
  </div>
);

export default Missions;